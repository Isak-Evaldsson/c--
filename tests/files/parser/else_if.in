int test() {
    if(1) {
        print(1);
    } else if(2) {
        print(2);
    } else {
        print(3);
    }
}