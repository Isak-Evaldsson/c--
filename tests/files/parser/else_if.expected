Prototype: test
    If:
        LiteralExpr: 1
    Do:
        FunCall: print
            LiteralExpr: 1
    Else:
        If:
            LiteralExpr: 2
        Do:
            FunCall: print
                LiteralExpr: 2
        Else:
            FunCall: print
                LiteralExpr: 3
